core/src/main/java/com/es/core/model/phone/JdbcPhoneDao.java

```diff
@@ -13,15 +13,33 @@
     @Resource
     private JdbcTemplate jdbcTemplate;
 
+    public void setJdbcTemplate(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
     public Optional<Phone> get(final Long key) {
-        throw new UnsupportedOperationException("TODO");
+        return Optional.ofNullable(jdbcTemplate.queryForObject("select * from phones where id=?",
```

Comment:
Usually dao queries are defined in String constants, better to use such approach, just hint